bentoml/__init__.py

```diff
@@ -73,8 +73,10 @@
     from bentoml import pytorch_lightning
 
     # Model management APIs
+    # Prometheus metrics client
     from . import io
     from . import models
+    from . import metrics
```

Comment:
Probably better to separate this and use `# isort` comment to avoid reformatting